asset_dashboard/static/js/components/BaseTable.js

```diff
@@ -1,51 +1,13 @@
 import React from 'react'
 import { useTable, usePagination, useSortBy } from 'react-table'
 
-const BaseTable = ({ rows = [],  getTrProps = props => props, rowClassNames, selector }) => {
+const BaseTable = ({ rows = [], columns, getTrProps = props => props, rowClassNames, sizeOfPage=15, pageSizeIncrements=[15, 30, 45] }) => {
```

Comment:
Made this component more generic so I can use it to show the asset search results. Before this, some details about the `ProjectTable` and `PortfolioTable` were hardcoded in here.